### PR TITLE
Anteneh Amare

### PR DESCRIPTION
**Pull request by dawit2119**

### CODE REVIEW

**Comment by ante-neh** [comment]
Anteneh Amare #2

**Comment by Dawit2119** [comment]
Anteneh Amare resolve for #2

**Comment by ante-neh** [comment]
Leet code

File: code_review.py [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (ante-neh) [inline code_review.py:1]: where is my name

Comment (ante-neh) [inline code_review.py:1]: where is my name 2

**Review by ante-neh — APPROVED** [review]

**Pull request merged**